graphdb_setup fails when `jq` cannot be installed

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Expected behavior

Stack launches and works correctly from recipes

### Current behavior

Stack launches (apparently correctly) but any request to the graph fails with 404 and

```json
{
  "detail": ": Unknown repository: my_db"
}
```

### Error message

```
Installing jq...
Reading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  libjq1 libonig5
The following NEW packages will be installed:
  jq libjq1 libonig5
0 upgraded, 3 newly installed, 0 to remove and 0 not upgraded.
Need to get 379 kB of archives.
After this operation, 1,125 kB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu noble/main amd64 libonig5 amd64 6.9.9-1build1 [172 kB]
Ign:2 http://archive.ubuntu.com/ubuntu noble-updates/main amd64 libjq1 amd64 1.7.1-3ubuntu0.24.04.1
Ign:3 http://archive.ubuntu.com/ubuntu noble-updates/main amd64 jq amd64 1.7.1-3ubuntu0.24.04.1
Err:2 http://security.ubuntu.com/ubuntu noble-updates/main amd64 libjq1 amd64 1.7.1-3ubuntu0.24.04.1
  404  Not Found [IP: 91.189.92.24 80]
Err:3 http://security.ubuntu.com/ubuntu noble-updates/main amd64 jq amd64 1.7.1-3ubuntu0.24.04.1
  404  Not Found [IP: 91.189.92.24 80]
Fetched 172 kB in 2s (107 kB/s)
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/j/jq/libjq1_1.7.1-3ubuntu0.24.04.1_amd64.deb  404  Not Found [IP: 91.189.92.24 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/j/jq/jq_1.7.1-3ubuntu0.24.04.1_amd64.deb  404  Not Found [IP: 91.189.92.24 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
Finished server setup.export FIRST_TIME_SETUP=on
Adding datasets to the database...

CLEARING EXISTING DATA FROM repositories/my_db...
[ERROR] 2026-04-28 01:59:03,583 [repositories/my_db | c.o.f.s.GraphDBProtocolExceptionResolver] Client sent bad request (404)
org.eclipse.rdf4j.http.server.ClientHTTPException: Unknown repository: my_db
```

### Environment

- OS: Ubuntu 22.04.5 LTS
- Python/Node version: 
- Docker version: Docker version 29.1.5, build 0e6fee6
- Docker Compose version: Docker Compose version v5.0.1

### How to reproduce

- Launch a production node stack from current main of recipes
- trace the logs of the graph

### Anything else?

Error likely introduced in 1bbbf6bcdbf5a815a0671d24bc57e2b70f179211 - when we switched from alpine based graphDB image to debian based. The alpine version included `apk add --no-cache jq` which **refreshes the package index** before installing. However the new version directly runs `apt-get install` **without refreshing the package index first**. Thus we are trying to install with a severely outdated package index - and that fails (either has just begun failing now, or we have just now tried to setup with the new recipe for the first time).

Either way, the fix is to first update the package index before installing

Sidenote: a more longterm fix is likely to remove the jq dependency entirely and instead either
-  rely on python + json package instead. 
- build a custom graph image with jq inside and ship that

--- a/scripts/graphdb_setup.sh
+++ b/scripts/graphdb_setup.sh
@@ -139,7 +139,7 @@ echo "The GraphDB server is being accessed at http://localhost:${NB_GRAPH_PORT}.
 # Install jq to construct JSON payloads for curl requests while preserving special characters
 apt-get update -qq
 echo "Installing jq..."
-apt-get install -y -q jq
+apt-get install -y -qq jq
 
 ##### First time GraphDB setup #####
 

--- a/scripts/graphdb_setup.sh
+++ b/scripts/graphdb_setup.sh
@@ -137,6 +137,7 @@ SCRIPT_DIR=$(dirname "$0")
 echo "The GraphDB server is being accessed at http://localhost:${NB_GRAPH_PORT}."
 
 # Install jq to construct JSON payloads for curl requests while preserving special characters
+apt-get update -qq
 echo "Installing jq..."
 apt-get install -y -q jq
 

